notifications.go

```diff
@@ -0,0 +1,271 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/go-telegram-bot-api/telegram-bot-api"
+	"io/ioutil"
+	"log"
+	"path/filepath"
+	"strconv"
+	"sync"
+)
+
+const (
+	notificationsDB = "notifications.json"
+)
+
+type notifications struct {
```

Comment:
Add comment to notifications struct (not strictly needed, but a good idea. Something like:
`// notifications holds the ...`